src/tracing/agent.cc

```diff
@@ -48,8 +48,7 @@ using v8::platform::tracing::TraceConfig;
 using v8::platform::tracing::TraceWriter;
 using std::string;
 
-Agent::Agent() {
-  tracing_controller_ = new TracingController();
+Agent::Agent() : tracing_controller_(new TracingController()) {
```

Comment:
If the `TracingController` life is 1:1 with the `Agent`, make it a member instead of a pointer-to.

Reply:
At this point, the lifetimes aren't perfectly aligned. After this PR lands, I have an intention to refactor & merge the `Agent` and `TracingController` concepts into a single structure.